docs/models.rst

```diff
@@ -76,14 +72,14 @@ function.  However the foreign key has to be separately declared with the
 
     class Person(db.Model):
         id = db.Column(db.Integer, primary_key=True)
-        name = db.Column(db.String(50))
-        addresses = db.relationship('Address', backref='person',
-                                    lazy='dynamic')
+        name = db.Column(db.String(50), nullable=False)
+        addresses = db.relationship('Address', backref='person', lazy=True)
 
     class Address(db.Model):
         id = db.Column(db.Integer, primary_key=True)
-        email = db.Column(db.String(50))
-        person_id = db.Column(db.Integer, db.ForeignKey('person.id'))
+        email = db.Column(db.String(120), nullable=False)
+        person_id = db.Column(db.Integer, db.ForeignKey('person.id'),
```

Comment:
really? interesting. didn't know about that.. not sure what i prefer there though, but maybe i'm just used to specifying it